### PR TITLE
new design by Viacheslav Rybak

### PR DESCRIPTION
**Pull request by RybakV**
new HTML design by Viacheslav Rybak

### CODE REVIEW

**Comment by RybakV** [comment]
removed .idea

File: index.html [hunk 1]
```diff
@@ -1,534 +1,414 @@
...
-    <!-- Google Font -->
-    <link href='https://fonts.googleapis.com/css?family=Raleway:500,600,700,800,900,400,300' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,900,300italic,400italic' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Droid+Sans+Mono' rel='stylesheet' type='text/css'>
-    <!-- Bootstrap -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <title>Devatlant</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /> 
+    <link rel='icon' href='./favicon.ico' type='image/x-icon' />
```
Comment (yev) [inline index.html]: please, use meta viewport tag - <meta name="viewport" content="width=device-width, initial-scale=1">

File: index.html [hunk 2]
```diff
@@ -1,534 +1,414 @@
...
-<html lang="en">
+<!DOCTYPE HTML>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="ru" lang="ru" dir="ltr">
 
 <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="devatlant - software company providing top-class solutions">
-    <meta name="keywords" content="software, company, Ukraine, France, frenchtech, startup, java, cloud, development, programming, Java, Spring, Maven, JPA, Testing, DevOps, Ansible">
```
Comment (yev) [inline index.html]: please, reuse the max from existing meta - to save info, icons etc

File: index.html [hunk 3]
```diff
@@ -1,534 +1,414 @@
...
-
-    <!-- Style -->
-    <link href="css/style.css" rel="stylesheet">
-    <link href="css/animate.css" rel="stylesheet">
-
-    <!-- Responsive CSS -->
-    <link href="css/responsive.css" rel="stylesheet">
-
-    <!-- Favicon -->
-    <link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
```
Comment (yev) [inline index.html]: we need this icons

**Pull request merged**